docs_user/adoption-attributes.adoc

```diff
@@ -4,27 +4,124 @@
 
 ifeval::["{build}" == "upstream"]
 :OpenShift: OpenShift
-:osp_prev_long: OpenStack
-:osp_acro: OSP
+:rhos_prev_long: OpenStack
+:rhos_acro: OSP
 :OpenStackShort: OSP
 :OpenShiftShort: OCP
-:osp_curr_ver: Antelope
-:osp_prev_ver: Wallaby
+:rhos_curr_ver: Antelope
+:rhos_prev_ver: Wallaby
 :OpenStackPreviousInstaller: TripleO
+
+//Components and services
+
+//Identity service (keystone)
+:identity_service_first_ref: Keystone
+:identity_service: Keystone
+
+//Shared File System service (manila)
 :rhos_component_storage_file_first_ref: Manila
 :rhos_component_storage_file: Manila
+
+//OpenStack Key Manager (barbican)
+:key_manager_first_ref: Barbican
+:key_manager: Barbican
+
+//OpenStack Networking service (neutron)
+:networking_first_ref: Neutron
+:networking: Neutron
+
+//Object Storage service (swift)
+:object_storage_first_ref: Swift
+:object_storage: Swift
+
+//Image service (glance)
+:image_service_first_ref: Glance
+:image_service: Glance
+
+//Compute service (nova)
+:compute_service_first_ref: Nova
```

Comment:
Hi Katie, the upstream service names all start with a capital in the PR, they should be lowercase, correct?

Reply:
Hmm...maybe the service names are the same for both upstream and downstream then. "Examples of these guidelines in service names are: the Block Storage service (cinder), the Object Storage service (swift), or the Bare Metal service (ironic)."

Reply:
Yes, it looks that way from those guidelines. 

"Developer documentation may refer to the project name, but end-user, operator, administrator, and application developer documentation must refer to the service name. When reviewing service names, consider the consumers of the information."